ArduCopter/Parameters.cpp

```diff
@@ -214,6 +214,20 @@ const AP_Param::Info Copter::var_info[] = {
     // @User: Standard
     GSCALAR(wp_yaw_behavior,  "WP_YAW_BEHAVIOR",    WP_YAW_BEHAVIOR_DEFAULT),
 
+    // @Param: AUTO_MAN_ALT
+    // @DisplayName: Auto Manual Altitude Control
+    // @Description: Flight mode developed by Aerospace Scannig Technologies AERSOCANTECH. Programmed by Systems Engineer ANDRES ARDILA SEDANO nick name a2sAndres.This flight mode is based on the Auto and Alt_hold flight modes, which allows manual altitude control.While entering the height control the aircraft stops.it re starts the forward advance by giving it pitch input and continuing with the flight plan
```

Comment:
This initial part of the description needs to be removed on both parameters. Also:

- needs spaces after a period
- this isn't a new flight mode, so description isn't accurate

Reply:
Revised and changed